src/InlineKeyboardPagination.php

```diff
@@ -2,6 +2,8 @@
 
 namespace TelegramBot\InlineKeyboardPagination;
 
+use JetBrains\PhpStorm\ArrayShape;
```

Comment:
Not entirely sure how I feel about these, but I'm open to giving it a try and keeping this in 😊

Reply:
Done :)